test/phabricator_tests/mock_phabricator.py

```diff
@@ -0,0 +1,350 @@
+            },
+            {
+                'userName': 'dummy_user-3',
+                'phid': 'PHID-USER-xxxxxxxxxxxxxxxxxxx3',
+                'realName': 'Dummy User',
+                'roles': [
+                    'verified', 'approved', 'activated'
+                ],
+                'image': 'userimage.com',
+                'uri': 'userurl.com'
+            }
+        ]
+        return user, raw_response
+
+    def mock_phabricator_user_search(username, phab):
+        return {
+            'cursor': {
+                'after': None,
+                'limit': 100,
+                'order': None,
+                'before': None
+            },
+            'maps': {},
+            'data': [
+                {
+                    'fields': {
+                        'username': 'dummyuser',
+                        'realName': 'Dummy User',
+                        'roles': [
+                            'verified',
+                            'approved',
+                            'activated'],
+                        'dateCreated': 1509530187,
+                        'policy': {
+                            'edit': 'no-one',
+                            'view': 'public'},
+                        'dateModified': 1509530333},
+                    'phid': 'PHID-USER-xxxxxxxxxxxxxxxxxxxx',
+                    'type': 'USER',
+                    'id': 86,
+                    'attachments': {}}],
+            'query': {
+                'queryKey': None
+            }
+        }
+
+    def mock_phabricator_raw_response():
+        return [
+            {
+                'userName': 'user1',
+                'phid': 'PHID-USER-xxxxxxxxxxxxxxxxxx',
+                'realName': 'user 1',
+                'roles': [
+                    'verified',
+                    'approved',
+                    'activated'
+                ],
+                'image': 'user1.image',
+                'uri': 'user1.url'
+            },
+            {
+                'userName': 'user2',
+                'phid': 'PHID-USER-xxxxxxxxxxxxxxxxxx',
+                'realName': 'user 2',
+                'roles': [
+                    'verified',
+                    'approved',
+                    'activated'
+                ],
+                'image': 'user2.image',
+                'uri': 'user2.url'
+            },
+            {
+                'userName': 'user3',
+                'phid': 'PHID-USER-test',
+                'realName': 'user 3',
+                'roles': [
+                    'verified',
+                    'approved',
+                    'activated'
+                ],
+                'image': 'user3.image',
+                'uri': 'user3.url'
+            }
+        ]
+
+    def mock_phabricator_update_interfaces():
+        return
+
+    def mock_phabricator_time_from_epoch(epoch_time):
+        return datetime.datetime.strptime('16Sep2012', '%d%b%Y')
+
+class InternetError(Exception):
```

Comment:
In a case that you would decide to remove the class and make functions instead, I would put the exception into `test_phabricator.py`.